packages/git-extractor/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "@sentry/node": "5.13.2",
+    "@types/base-64": "^0.1.3",
```

Comment:
Shouldn't this be in `devDependencies`? 🤔

Reply:
You it should 😄